src/site/content/en/handbook/effective-instruction/index.md

```diff
@@ -5,22 +5,22 @@ authors:
   - mfriesenhahn
 date: 2019-06-26
 description: |
-  Strategies for writing effective instruction on web.dev, especially for adult learners.
+  Learn strategies for writing effective instruction on web.dev, especially for adult learners.
 ---
 
 Most web.dev content includes at least some instruction. These guidelines help make your instruction easy to follow.
 
-Explain _why_ to do something before explaining _how_ to do it. Knowing what they're trying to accomplish helps readers understand the purpose of individual steps.
+Explain _why_ to do something before explaining _how_ to do it. Knowing the end goal helps readers understand the purpose of individual steps.
 
 Keep the scope of instruction manageable. If a task is complex, find ways to chunk the steps. For example, provide the essential pieces in a post and put the ancillary pieces in one or more codelabs.
```

Comment:
The for example bit doesn't tell me more. I'm still not sure what info is essential vs ancillary. Can we have a practical example of good/bad?

Also, ancillary seems like an overly complex word.

Reply:
'end goal' -> 'goal' (Shrunk and White II.17: Omit needless words.)

Reply:
I'm not sure that we can or should provide criteria for determining whether content is supplemental to the main idea of a piece of content. Seems like those criteria would be highly dependent on context? Did you have some in mind? Or are you leaning toward cutting this as unhelpfully general?